src/main/java/org/sagebionetworks/web/client/widget/table/v2/results/TableQueryResultWikiWidget.java

```diff
@@ -128,13 +118,8 @@ public void onSuccess(EntityBundle bundle) {
 				boolean isCurrentVersion = true;
 				entityActionController.configure(actionMenu, bundle, isCurrentVersion, bundle.getRootWikiId(), EntityArea.TABLES);
 				boolean canEdit = false;
-				if (!DisplayUtils.isInTestWebsite(ginInjector.getCookieProvider())) {
-					getTableEntityWidget().configure(bundle, tableVersionNumber, canEdit, TableQueryResultWikiWidget.this, actionMenu);
-					hideEditActions();
-				} else {
-					hideEditActions();
-					getTableEntityWidgetV2().configure(bundle, tableVersionNumber, canEdit, isShowTableOnly, TableQueryResultWikiWidget.this, actionMenu);
-				}
+				hideEditActions();
+				getTableEntityWidgetV2().configure(bundle, tableVersionNumber, canEdit, isShowTableOnly, TableQueryResultWikiWidget.this, actionMenu);
```

Comment:
@jay-hodgson Looks like we're still properly handling the special case for wiki

Reply:
Could you also take a look at TableQueryResultWikiViewImpl.  The isShowTableOnly option should be available on release of Explore 2.0